### PR TITLE
Game as a Widget

### PR DESCRIPTION
**Pull request by feroult**
Deprecating the `new Game().widget` syntax.

### CODE REVIEW

File: lib/game.dart [hunk 1]
```diff
@@ -36,23 +38,24 @@ abstract class Game {
...
-  Widget get widget {
-    if (_widget == null) {
-      _widget = new Center(
-          child: new Directionality(
-              textDirection: TextDirection.ltr,
-              child: new _GameRenderObjectWidget(this)));
-    }
-    return _widget;
-  }
+  @Deprecated('Use Game is a Widget')
```
Comment (erickzanardo) [inline lib/game.dart]: Use game as a Widget?

Comment (feroult) [reply lib/game.dart]: FIxed :)

**Comment by luanpotter** [comment]
This breaks (or rather deprecates) all the examples and docs, we have to fix those!

**Comment by feroult** [comment]
Ok! Let's do that. But what do you think? Is it better to rollback to the `.widget` api?

**Comment by luanpotter** [comment]
No, let's leave it as that, LGTM. But keeping the old deprecated for now, when we release 0.11 we can remove the deprecated version :)

**Comment by erickzanardo** [comment]
Should web release `0.10.3` soon? I am thinking on resolving #45 and I think would be good to create this doc already contemplating this changes. What do you think?

**Comment by feroult** [comment]
Agree.

**Comment by mtcliatt** [comment]
What was the motivation for this change? (just curious)

**Comment by erickzanardo** [comment]
I think it is because it is more semantic correctly. I mean, the way you add a Flame game was by adding a widget in the Flutter app. A GameWidget seems more easy to understand, than a property widget stored inside Game class.

**Pull request merged**